loopin.go

```diff
@@ -222,6 +222,27 @@ func resumeLoopInSwap(reqContext context.Context, cfg *swapConfig,
 	return swap, nil
 }
 
+func createHtlcs(swapKit *swapKit) (
```

Comment:
If this is a method on `loopInSwap`, you can directly assign the htlcs to fields. It becomes more like `swap.initHtlcs()`

Reply:
done